src/app/CMakeLists.txt

```diff
@@ -100,6 +156,62 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Darwin")
         ${qBittorrent_SOURCE_DIR}/dist/mac/qt.conf
         ${qBittorrent_SOURCE_DIR}/dist/mac/qBitTorrentDocument.icns
         ${qBittorrent_SOURCE_DIR}/dist/mac/qbittorrent_mac.icns
+        ${qBittorrent_SOURCE_DIR}/dist/mac/lproj/ar.lproj
```

Comment:
I believe there should be a way to make it in a single entry using some wildcard.
Moreover, there may even be a way to create all these folders without storing empty folders in the resources.